api-catalog-ui/frontend/config-overrides.js

```diff
@@ -13,6 +13,11 @@ module.exports = {
 
         config.module.rules = [
             ...config.module.rules,
+            {
+                test: /\.html$/i,
+                use: 'html-loader',
+                // loader: 'html-loader',
```

Comment:
is this comment important?

Reply:
I had forgotten to delete it, thanks for noticing, removed.